tables/tr-g2.tbl

```diff
@@ -0,0 +1,17 @@
+#+locale:tr
+#+type:literary
+#+contraction:full
+#+grade:2
+#+dots:8
+
+# TODO: Please correct the metadata above. It is not meant to be
```

Comment:
I'm not the original author of the tr-g2.ctb file (as the copyright states), I'm only the author of the .tbl file, so I can't be entirely sure that it's correct.